OpenTaiko/src/Common/CConfigIni.cs

```diff
@@ -2662,821 +2617,924 @@ public void t書き出し(string iniファイル名) {
-														this.ReadAndSetKey(str4, this.KeyAssign.Drums.TrainingDecreaseScrollSpeed);
-														break;
-													}
-												case "TrainingIncreaseSongSpeed": {
-														this.ReadAndSetKey(str4, this.KeyAssign.Drums.TrainingIncreaseSongSpeed);
-														break;
-													}
-												case "TrainingDecreaseSongSpeed": {
-														this.ReadAndSetKey(str4, this.KeyAssign.Drums.TrainingDecreaseSongSpeed);
-														break;
-													}
-												case "TrainingToggleAuto": {
-														this.ReadAndSetKey(str4, this.KeyAssign.Drums.TrainingToggleAuto);
-														break;
-													}
-												case "TrainingBranchNormal": {
-														this.ReadAndSetKey(str4, this.KeyAssign.Drums.TrainingBranchNormal);
-														break;
-													}
-												case "TrainingBranchExpert": {
-														this.ReadAndSetKey(str4, this.KeyAssign.Drums.TrainingBranchExpert);
-														break;
-													}
-												case "TrainingBranchMaster": {
-														this.ReadAndSetKey(str4, this.KeyAssign.Drums.TrainingBranchMaster);
-														break;
-													}
-												case "TrainingPause": {
-														this.ReadAndSetKey(str4, this.KeyAssign.Drums.TrainingPause);
-														break;
-													}
-												case "TrainingBookmark": {
-														this.ReadAndSetKey(str4, this.KeyAssign.Drums.TrainingBookmark);
-														break;
-													}
-												case "TrainingMoveForwardMeasure": {
-														this.ReadAndSetKey(str4, this.KeyAssign.Drums.TrainingMoveForwardMeasure);
-														break;
-													}
-												case "TrainingMoveBackMeasure": {
-														this.ReadAndSetKey(str4, this.KeyAssign.Drums.TrainingMoveBackMeasure);
-														break;
-													}
-												case "TrainingSkipForwardMeasure": {
-														this.ReadAndSetKey(str4, this.KeyAssign.Drums.TrainingSkipForwardMeasure);
-														break;
-													}
-												case "TrainingSkipBackMeasure": {
-														this.ReadAndSetKey(str4, this.KeyAssign.Drums.TrainingSkipBackMeasure);
-														break;
-													}
-												case "TrainingJumpToFirstMeasure": {
-														this.ReadAndSetKey(str4, this.KeyAssign.Drums.TrainingJumpToFirstMeasure);
-														break;
-													}
-												case "TrainingJumpToLastMeasure": {
-														this.ReadAndSetKey(str4, this.KeyAssign.Drums.TrainingJumpToLastMeasure);
-														break;
-													}
-											}
-											continue;
-										}
-									//-----------------------------
-									#endregion
-									case ESectionType.DEBUG: {
-											switch(str3) {
-												case "ImGui": {
-														this.DEBUG_bShowImgui = CConversion.bONorOFF(str4[0]);
-														break;
-													}
-											}
-											continue;
-										}
-								}
+				string line = s.Replace('\t', ' ').TrimStart(new char[] { '\t', ' ' });

```

Comment:
Should warp per-call because each call contains comma-separated values.

Reply:
I'm not sure what to do here. Do you want to place `TrimStart` on the next LOC?

Reply:
```csharp
string line = s.Replace('\t', ' ')
      .TrimStart(new char[] { '\t', ' ' });
```
`a.b(c).d(e)` and on would be warped as `a.b(c)\n.d(e)`

Reply:
Okay, then I understood you correctly. I'll adjust.